spec/commands/decidim/solutions/admin/update_solution_spec.rb

```diff
@@ -136,9 +136,9 @@ module Admin
               expect(translated(solution.title)).to eq title
             end
 
-            it "sets challenge" do
+            it "challenge is nil" do
               subject.call
-              expect(solution.challenge).to eq(challenge)
+              expect(solution.challenge).to eq nil
```

Comment:
we don't want to change the test, it is the other way around. The test must be fixed because when the problem is not active there must be a challenge...

Reply:
What happens is that if there is no problem, a challenge cannot be associated with the solution from the BO. The associated challenge must be that of the problem...